src/encoded/static/components/matrix_experiment.js

```diff
@@ -106,7 +106,12 @@ const convertExperimentToDataTable = (context, getRowCategories, getRowSubCatego
         colTitleMap[colCategoryBucket.key] = colIndex;
         return colCategoryBucket.key;
     });
-    const searchUrl = [context.search_base.split('&')[0], '&status=released'].join(''); // fancy way of removing query string parameters from URL
+
+    // This gets the statues and the base url and combines them
+    const urlComponents = context.search_base.split('&');
+    const statues = urlComponents.filter(component => component.includes('status')).join('&');
+    const searchUrl = `${urlComponents[0]}${statues ? ['&', statues].join('') : ''}`;
```

Comment:
I think you meant the const to be `status` and not `statues`?

What error did you get? This works for me:
```
const searchUrl = `${urlComponents[0]}${statues ? `&${statues}` : ''}`;
```

Reply:
@forresttanaka  There can be multiples statues as a user can select "status=released" AND "status=archived". So I used the variable names "statuses".

You care correct on the string template. I probably improperly used a tick.

Reply:
@forresttanaka This code was removed. I also had to rebase on origin/dev

Reply:
OK, then you meant “statuses” instead of “statues” — perfectly good word, but not the one you wanted.

Reply:
this code is gone now.